### PR TITLE
Refactored seed script.

### PR DESCRIPTION
**Pull request by theofficetroll**
https://trello.com/c/Dl19xHyG

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -10,21 +10,21 @@ const seedBundles = () => {
...
-    tierId3 = i*3 + 3;
+    tierId1 = i * 3 + 1;
+    tierId2 = i * 3 + 2;
+    tierId3 = i * 3 + 3;
     bundleData.push({
-      bundleId : bundleId,
-      tierId : tierId1,
+      bundleId: bundleId,
+      tierId: tierId1,
       tierOrder: 1
```
Comment (gektin) [inline database/seed.js:19]: I believe we are supposed to use trailing commas per the airbnb style guide. See also, e.g., lines 24, 28, 48, 54, 61, and 69.

File: database/seed.js [hunk 2]
```diff
@@ -42,54 +42,86 @@ const seedTiers = () => {
...
+        console.log('tiers created');
+      })
+      .catch(err => {
+        console.log('failed to create tiers', err);
+      })
+    })
+  });
+};
 
+seed();
```
Comment (gektin) [inline database/seed.js:124]: This reorganization (i.e., taking this out of index and putting it in seed) makes a ton of sense to me.

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "",
     "test": "jest --detectOpenHandles",
-    "seed": "node database/index.js",
+    "seed": "node database/seed.js",
```
Comment (gektin) [inline package.json:9]: Great. 👍

**Review by gektin — APPROVED** [review]
These simple changes make sense to me. I don't have any real comments, other than to confirm that you've tested this by dropping your database/table and running the seed script.

File: database/seed.js (hunk 1 above)
Comment (theofficetroll) [reply database/seed.js:19]: Thank you.

File: database/seed.js (hunk 2 above)
Comment (theofficetroll) [reply database/seed.js:124]: Yeah, I'd been planning to move it over once I did my refactor, but hadn't gotten to it until now.

**Pull request merged**